src/app/nation/nation.component.ts

```diff
@@ -94,4 +95,15 @@ export class NationComponent implements OnInit {
     this.createForm();
   }
 
+  view(id) {
+    if(id){
+      this.couchService.get('nations/' + id)
```

Comment:
this data will be taken from another couchdb. which will be another url. so try to fix the link of getting nation detail. Planet will not have nation detail.